src/main.js

```diff
@@ -207,21 +207,21 @@ saveViewMainButton.addEventListener('click', renderMainPage);
 
 makePosterButton.addEventListener('click', saveMyPoster);
 
-savePosterButton.addEventListener('click', savePosterActions); //NEW
+savePosterButton.addEventListener('click', savePosterActions);
 
 savedPosterGrid.addEventListener('dblclick', deleteSavedPoster);
 
 // functions and event handlers go here 👇
 
 function createPoster() {
-  var imageRandomNumber = getRandomIndex(images); //NEW
+  var imageRandomNumber = getRandomIndex(images);
   currentPoster = new Poster(images[imageRandomNumber].url, images[imageRandomNumber].altText, titles[getRandomIndex(titles)], quotes[getRandomIndex(quotes)]);
   renderCoverImage(currentPoster);
```

Comment:
It looks like you've added an argument here of currentPoster but don't have a parameter set in your renderCoverImage function

Reply:
@robertjosephderouin We can delete the parameter in the renderCoverImage function for currentPoster. This was the idea that we might eliminate the global variables but we decided not to do that based on instructor feedback. If you can make that change, I'll merge it, and then I'll send a note to Hannah.